src/editor.js

```diff
@@ -478,6 +567,24 @@ class Editor {
     assert(false, `I don't know how to toggle "${name}".`);
   }
 
+  async loadHtml_({target: {dataset}}) {
+    const name = assert(dataset.name);
+    const contentResponse = await fetch(
+      (await this.state_.templates)[name].contentUrl
```

Comment:
1. Let's destructure `contentUrl` and `files` here for style and to limit the amount of `await`s.
2. Let's rename `name` to `templateName` for inner `files` loop.

```js
async selectTemplate_({target: {dataset}}) {
  const templateName = assert(dataset.name);
  const {contentUrl, files} = (await this.state_.templates)[name];
  const contentResponse = await successfulFetch(this.win, contentUrl);

  this.codeMirror_.setValue(await contentResponse.text());

  this.state_.files = files.map(name => ({
    name,
    url: this.getTemplateFileUrl_(templateName, name),
  }));

  this.state_.isFilesPanelDisplayed = true;
  this.state_.isTemplateLightboxDisplayed = false;

  this.updatePreview_();
}
```